rpc/blockchain.go

```diff
@@ -708,3 +708,11 @@ func isBlockGreaterThanLatest(hmy *hmy.Harmony, blockNum rpc.BlockNumber) bool {
 	}
 	return uint64(blockNum) > hmy.CurrentBlock().NumberU64()
 }
+
+func (s *PublicBlockchainService) GetCurrentNodeBackupState(ctx context.Context) (bool, error) {
```

Comment:
would it be better to integrate the node state with getNodeMetadata rpc to reduce the number of supported RPC?